File: 9-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    if (x < 0){
        return false
    }

    x = x.toString()

    for(i = 0; i < (x.length/2).toFixed(0); i++){
        if (x[i] != x[x.length-i-1]){
            return false
        }
    }
    
    return true
};

console.log(isPalindrome(10))

File: python/9-palindrome-number.py
def isPalindrome(x):
    """
    :type x: int
    :rtype: bool
    """
    if x < 0:
        return False

    x = str(x)
    
    print(x[::-1])
    
    if x == x[::-1]:
        return True
    else:
        return False

print(isPalindrome(10))